### PR TITLE
Update aarch64_linux build to Jetpack 4.4 and CUDA 10.2

### PR DESCRIPTION
**Pull request by JanuszL**
- changes the build image for aarch64_linux to Ubuntu 18.03 and CUDA 10.2
- updates Jetpack version to 4.4
- Jetpack 4.4 is in the registration-required zone so ask user to download it first before the build - similar to QNX

Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

#### Why we need this PR?
*Pick one, remove the rest*
- It updates aarch64_linux build to Jetpack 4.4 and CUDA 10.2

#### What happened in this PR?
*Fill relevant points, put NA otherwise. Replace anything inside []*
 - What solution was applied:
     changes the build image for aarch64_linux to Ubuntu 18.03 and CUDA 10.2
     updates Jetpack version to 4.4
    Jetpack 4.4 is in the registration-required zone so ask user to download it first before the build - similar to QNX
 - Affected modules and functionalities:
     docker build for aarch64_linux
 - Key points relevant for the review:
     NA
 - Validation and testing:
     CI
 - Documentation (including examples):
     build procedure docs

**JIRA TASK**: *[NA]*

### CODE REVIEW

File: docs/compilation.rst [hunk 1]
```diff
@@ -339,12 +339,21 @@ Cross-compiling DALI C++ API for aarch64 Linux (Docker)
   x86-64 target and they are turned off in this build. There is no support for DALI Python library
   on aarch64 yet. Some Operators may not work as intended due to x86-64 specific implementations.
 
+Setup
+^^^^^
+Using the SDK Manager for NVIDIA Jetson as the product category adn JetPack 4.4.
```
Comment (klecki) [inline docs/compilation.rst:344]:
```suggestion
Download the JetPack 4.4 SDK for NVIDIA Jetson using the SDK Manager, following the instruction provided here: https://developer.nvidia.com/embedded/jetpack 
```

File: docs/compilation.rst [hunk 2]
```diff
@@ -339,12 +339,21 @@ Cross-compiling DALI C++ API for aarch64 Linux (Docker)
...
+Then select CUDA for the host. After download process has been completed move
+``cuda-repo-ubuntu1804-10-2-local-10.2.89-440.40_1.0-1_amd64.deb`` and
+``cuda-repo-cross-aarch64-10-2-local-10.2.89_1.0-1_all.deb`` from the download folder
+to main DALI folder.
```
Comment (klecki) [inline docs/compilation.rst:348]: Maybe mention that they are reqired for the cross-build?

File: docs/compilation.rst (hunk 1 above)
Comment (JanuszL) [reply docs/compilation.rst]: Done

File: docs/compilation.rst (hunk 2 above)
Comment (JanuszL) [reply docs/compilation.rst]: Done

**Pull request merged**